pynumdiff/finite_difference/__init__.py

```diff
@@ -1,5 +1,5 @@
 """This module implements some common finite difference schemes
 """
-from ._finite_difference import first_order, second_order, fourth_order
+from ._finite_difference import finite_difference, first_order, second_order, fourth_order
```

Comment:
There are simply more public methods now. The docstring makes reference to the fact other methods in the module really just call the backing one.